test/verify.test.js

```diff
@@ -77,7 +77,7 @@ describe('passport verify', function() {
       get: function () {
         return {
           load: function () {
-            return q(null);
+            return new q(() => {});
```

Comment:
can be replaced by `return Promise.resolve({})` to remove the dependency with `bluebird`